doc/lpvs-api.yaml

```diff
@@ -48,6 +48,55 @@ paths:
               schema:
                 $ref: '#/components/schemas/WebhookResponseForbidden'
 
+  /scan/{gitHubDomain}/{gitHubOrg}/{gitHubRepo}/{prNumber}:
```

Comment:
I don't think that `gitHubDomain` is necessary. It is already set to 'github.api.url' and if detailed regulations are not provided, errors are likely to happen.

```suggestion
  /scan/{gitHubOrg}/{gitHubRepo}/{prNumber}:
```

Reply:
I agree with you. The code is updated.